citations-tool/tool/src/webapp/vm/citation/create.vm

```diff
@@ -376,14 +376,15 @@
 	 */
 	function trackPreferredUrl( checkboxId )
 	{
-		// has this been clicked?
-		if( $( "#" + checkboxId ).attr( "checked" ) != null )
+		var checkBoxClicked = $( "#" + checkboxId );
+		// has this been checked?
+		if( checkBoxClicked.attr( "checked" ) == true )
```

Comment:
you are right, just checked jquery documentation, pre 1.6 jquery version returns boolean for $( elem ).attr( "checked" ) and on this page we are loading jquery1.4. will correct it